### ISSUE TITLE
vanillafix150 + liteloader don't open game

### ISSUE THREAD

**Issue by Huaxidesu** [DimensionalDevelopment/VanillaFix#253]
latest.log

>[23:45:15] [main/INFO]: Loading tweak class name net.minecraftforge.fml.common.launcher.FMLTweaker
[23:45:15] [main/INFO]: Using primary tweak class name net.minecraftforge.fml.common.launcher.FMLTweaker
[23:45:15] [main/INFO]: Loading tweak class name com.mumfrey.liteloader.launch.LiteLoaderTweaker
[23:45:15] [main/INFO]: Loading tweak class name optifine.OptiFineForgeTweaker
[23:45:15] [main/INFO]: Calling tweak class net.minecraftforge.fml.common.launcher.FMLTweaker
[23:45:15] [main/INFO]: Forge Mod Loader version 14.23.5.2847 for Minecraft 1.12.2 loading
[23:45:15] [main/INFO]: Java is Java HotSpot(TM) 64-Bit Server VM, version 1.8.0_241, running on Windows 10:amd64:10.0, installed at C:\Program Files\Java\jdk1.8.0_241\jre
[23:45:15] [main/WARN]: The coremod FMLCorePlugin (net.minecraftforge.fml.relauncher.FMLCorePlugin) is not signed!
[23:45:15] [main/WARN]: The coremod FMLForgePlugin (net.minecraftforge.classloading.FMLForgePlugin) is not signed!
[23:45:15] [main/INFO]: Searching F:\HyperCol\ttt\.minecraft\mods for mods
[23:45:15] [main/INFO]: Searching F:\HyperCol\ttt\.minecraft\mods\1.12.2 for mods
[23:45:15] [main/INFO]: Loading tweaker org.spongepowered.asm.launch.MixinTweaker from VanillaFix-1.0.10-150.jar
[23:45:15] [main/INFO]: Calling tweak class com.mumfrey.liteloader.launch.LiteLoaderTweaker
[23:45:15] [main/INFO]: Bootstrapping LiteLoader 1.12.2
[23:45:15] [main/INFO]: Registering API provider class com.mumfrey.liteloader.client.api.LiteLoaderCoreAPIClient
[23:45:15] [main/INFO]: Spawning API provider class 'com.mumfrey.liteloader.client.api.LiteLoaderCoreAPIClient' ...
[23:45:15] [main/INFO]: API provider class 'com.mumfrey.liteloader.client.api.LiteLoaderCoreAPIClient' provides API 'liteloader'
[23:45:15] [main/INFO]: Initialising API 'liteloader' ...
[23:45:15] [main/ERROR]: Catching
org.spongepowered.asm.service.ServiceInitialisationException: ModLauncher is not available
	at org.spongepowered.asm.service.modlauncher.MixinServiceModLauncherBootstrap.bootstrap(MixinServiceModLauncherBootstrap.java:52) ~[VanillaFix-1.0.10-150.jar:?]
	at org.spongepowered.asm.service.MixinService.runBootServices(MixinService.java:79) [liteloader-1.12.2-20171128.144431-4.jar:1.12.2-SNAPSHOT+jnks-b4.git-4cc2bb00c8eff08863c9f7625be08983951546a5]
	at org.spongepowered.asm.service.MixinService.<init>(MixinService.java:72) [liteloader-1.12.2-20171128.144431-4.jar:1.12.2-SNAPSHOT+jnks-b4.git-4cc2bb00c8eff08863c9f7625be08983951546a5]
	at org.spongepowered.asm.service.MixinService.getInstance(MixinService.java:92) [liteloader-1.12.2-20171128.144431-4.jar:1.12.2-SNAPSHOT+jnks-b4.git-4cc2bb00c8eff08863c9f7625be08983951546a5]
	at org.spongepowered.asm.service.MixinService.boot(MixinService.java:102) [liteloader-1.12.2-20171128.144431-4.jar:1.12.2-SNAPSHOT+jnks-b4.git-4cc2bb00c8eff08863c9f7625be08983951546a5]
	at org.spongepowered.asm.launch.MixinBootstrap.<clinit>(MixinBootstrap.java:76) [liteloader-1.12.2-20171128.144431-4.jar:1.12.2-SNAPSHOT+jnks-b4.git-4cc2bb00c8eff08863c9f7625be08983951546a5]
	at com.mumfrey.liteloader.launch.LiteLoaderTweaker.onPreInit(LiteLoaderTweaker.java:383) [liteloader-1.12.2-20171128.144431-4.jar:1.12.2-SNAPSHOT+jnks-b4.git-4cc2bb00c8eff08863c9f7625be08983951546a5]
	at com.mumfrey.liteloader.launch.LiteLoaderTweaker.acceptOptions(LiteLoaderTweaker.java:285) [liteloader-1.12.2-20171128.144431-4.jar:1.12.2-SNAPSHOT+jnks-b4.git-4cc2bb00c8eff08863c9f7625be08983951546a5]
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:114) [launchwrapper-1.12.jar:?]
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28) [launchwrapper-1.12.jar:?]
[23:45:15] [main/INFO]: SpongePowered MIXIN Subsystem Version=0.7.5 Source=file:/F:/HyperCol/%e5%ae%a2%e6%88%b7%e7%ab%af/ttt/.minecraft/libraries/com/mumfrey/liteloader/1.12.2-20171128.144431-4/liteloader-1.12.2-20171128.144431-4.jar Service=LaunchWrapper Env=CLIENT
[23:45:16] [main/INFO]: Initializing VanillaFix
[23:45:16] [main/INFO]: Initializing StacktraceDeobfuscator
[23:45:16] [main/INFO]: Downloading MCP method mappings to: methods-stable_39.csv

**Comment by Star-Craft** [comment]
liteloader uses an old version of mixin(0.7.5)

**Comment by BoogieMonster1O1** [comment]
Read the pinned issues (#265 and #296)
tl;dr - rename `VanillaFix-1.0.10-150.jar` to `_VanillaFix-1.0.10-150.jar`
